app/assets/stylesheets/local/faq.scss

```diff
@@ -0,0 +1,10 @@
+.faq-page {
+  dl {
+    dt {
+      margin-top: 1em;
+    }
+    dd {
+      margin-top: .25em;
+    }
+  }
+}
```

Comment:
I'm not sure whether there are other DLs used for metadata, e.g. I'll use a common class, though.